keyboards/ymd96/info.json

```
@@ -0,0 +1,19 @@
+{
+    "keyboard_name": "ymd96", 
+    "url": "", 
+    "maintainer": "Andrew", 
```

This should actually be the GitHub username of the keyboard's maintainer.

Reference: https://docs.qmk.fm/#/hardware_keyboard_guidelines?id=infojson-format

```suggestion
    "maintainer": "sparkyman215", 
```